lazer/contracts/evm/src/PythLazer.sol

```diff
@@ -6,6 +6,7 @@ import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 
 contract PythLazer is OwnableUpgradeable, UUPSUpgradeable {
     TrustedSignerInfo[2] public trustedSigners;
+    uint256 public verification_fee = 0.00001 ether;
```

Comment:
Not sure what is an appropriate fee. Changing this value does not impact the tests (Unless the fee is a fractional wei which I don't think is possible?) so we can just change this to whatever we want.

Reply:
Is this 1 wei? I thought 1 eth was 10^18 wei